### PR TITLE
added partial support for TRACE_S2MM_FULL (AIE flows only)

### PR DESCRIPTION
**Pull request by pgschuey**

### CODE REVIEW

File: src/runtime_src/core/include/xclbin.h [hunk 1]
```diff
@@ -313,6 +313,7 @@ extern "C" {
         AXI_STREAM_MONITOR,
         AXI_STREAM_PROTOCOL_CHECKER,
         TRACE_S2MM,
+        TRACE_S2MM_FULL,
```
Comment (rozumx) [inline src/runtime_src/core/include/xclbin.h]: Being that the enumeration values are inserted into the xclbin archive, shouldn't TRACE_S2MM_FULL be added to the end of the enumeration list.  

Currently how the code is implemented, past AXI_DMA enumeration values contained in a previous xclbin archive are now mapped onto TRACE_S2MM_FULL, which I don't believe is the desired behavior.

Comment (pgschuey) [reply src/runtime_src/core/include/xclbin.h]: moved new enum value to end as requested

**Comment by pgschuey** [comment]
Please retest

**Comment by NaAl-Xlnx** [comment]
retest this please

**Pull request merged**